src/cljc/ataru/util.cljc

```diff
@@ -68,3 +69,16 @@
     (->> dropdown-options
         (filter (comp (partial = value) :value))
       (mapcat :followups))))
+
+(def ^:private b-limit 1024)
+(def ^:private kb-limit 102400)
+
+(defn size-bytes->str [bytes]
+  #?(:cljs (condp > bytes
+             b-limit (str bytes "B")
+             kb-limit (gstring/format "%.01fkB" (/ bytes 1024))
```

Comment:
The template parameters are a bit different too, which is why I didn't do it. But yeah, I can do that with `partial`. I'll try that.